kroki/plugin.py

```diff
@@ -95,7 +96,7 @@ def _save_kroki_image_and_get_url(self, file_name, image_data, files, page):
         mkdocs_file = File(get_url, self._tmp_dir.name, self._output_dir, False)
         files.append(mkdocs_file)
 
-        return {mkdocs_file.url_relative_to(page.file)}
```

Comment:
Here's the issue. This shouldn't be in curly braces.

Reply:
No need to undo #20 It's just the two characters out of place.

Reply:
Hi @ITopiaJesseGoerzen ,

this exactly was my fist attempt, but it broke the `site_url` usecase in my environment. :unamused: 

This project really needs tests...

Reply:
I'd have to agree. Sorry for late reply. I've been busy.

Hmm... then, maybe this works?:

``` python
f"{mkdocs_file.url_relative_to(page.file)}"
```

Relative URLs are highly preferred across the board. Absolute urls would break the site when viewing as static files and when hosting the site on a sub-path.